membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -36,12 +36,37 @@ class AccountController extends LazyLogging {
   lazy val mmaAction = NoCacheAction andThen mmaCorsFilter andThen BackendFromCookieAction
   lazy val mmaCardAction = NoCacheAction andThen corsCardFilter andThen BackendFromCookieAction
 
+
+  def cancelAccount [P <: SubscriptionPlan.AnyPlan : SubPlanReads] = mmaCardAction.async { implicit request =>
```

Comment:
shouldn't this be cancelSubscription or something like that?

Reply:
Yeap